cava/Cava2SystemVerilog.hs

```diff
@@ -106,17 +106,32 @@ instanceArgs inst
       Netlist.Xorcy li ci o -> [o, ci, li]
       Netlist.Muxcy s di ci o -> [o, ci, di, s]
 
-generateInstance :: Netlist.Instance -> String
-generateInstance (Netlist.Coq_mkInstance number (Netlist.DelayBit i o))
+instanceNumber :: Netlist.Primitive -> Integer
+instanceNumber inst
```

Comment:
So this is the instance number for gates with just one output. Something else will need to be done for gates with more than one bit of output.

Reply:
Yes this Haskell would need to change, although the Primitive definition doesn't preclude multiple outputs.